### DISCUSSION TITLE
Auto hide keyboard not working in MAUI as in Xamarin

### DISCUSSION THREAD

**Discussion by JoacimWall** [dotnet/maui#17997] [Q&A]
In Xamarin when the keyboard show when you focus a entry field you can just klick outside and it unfocused and hides the keyboard this is not working in MAUI. Is there a way to get this behavour back.  

https://github.com/dotnet/maui/assets/9278908/05adaac8-c40d-4189-af30-1c2ebf4028b3

**Comment by Engisan** [comment]
I second that, this is a big problem for me as well.
